### DISCUSSION TITLE
WIN 11 It's mandatory??

### DISCUSSION THREAD

**Discussion by Jamir-boop** [soulhotel/FF-ULTIMA#27] [Q&A] [answered]
I use win 10 as my daily OS, tried installing this theme but it dosen't work. 🤔🤔

**Comment by Kaedriz** [comment]
Can you describe how you installed it?
And post a screenshot how your `chrome` folder looks like, so we can diagnose problem.

It should work on Windows 11 as well as 10, as there is none Windows specific functionality.
Just be sure you followed installation instructions correctly.

**Comment by Jamir-boop** [comment]
It's fixed, I was on the wrong profile folder. (I'm using Firefox developer) Apparently it has a different default profile folder than regular Firefox.

**Reply by Kaedriz** [reply, marked as answer]
Yes, you should be using `*.defaul-release` or `*.default` one for normal Firefox,
`*.dev-edition-default` for Firefox Dev,
`*.default-nightly` for Firefox Nightly.
Or something along these lines, as Firefox naming is strange. Just localize your profile folder in ways written in instructions, it will point you directly.

But bear in mind that incoming feature in Firefox Dev/Nightly might break some parts of theme.
